app/pods/publications/publication/translations/route.js

```diff
@@ -1,5 +1,24 @@
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import Route from '@ember/routing/route';
+import { hash } from 'rsvp';
 
-export default class PublicationTranslationsRoute extends Route.extend(AuthenticatedRouteMixin) {
+export default class PublicationDocumentsRoute extends Route.extend(AuthenticatedRouteMixin) {
+  async model() {
+    const publicationFlow = this.modelFor('publications.publication');
+    let _case = await publicationFlow.get('case');
+    _case = await this.store.findRecord('case', _case.get('id'), {
+      include: 'pieces,pieces.document-container,pieces.document-container.type',
+      reload: true,
+    });
+    const subcases = await this.store.query('subcase', {
+      // include: 'activity',
```

Comment:
Als het hier in comment staat, niet nodig en gewoon wegdoen?

Reply:
Dit is al voorbereidend werk voor volgende story.
Dit gaat er moeten staan maar de api hangt hier voorlopig nog op. Reden ombekend.

Reply:
Dit mag zeker blijven staan en moet nog opgelost worden. Zij het in een andere story.